pkg/get/get_test.go

```diff
@@ -3354,7 +3372,7 @@ func Test_DownloadKrew(t *testing.T) {
 				t.Fatal(err)
 			}
 			if got != tc.url {
-				t.Errorf("want: %s, got: %s", tc.url, got)
+				t.Errorf("want: %s for %s on %s, got: %s", tc.url, tc.os, tc.arch, got)
```

Comment:
Added additional detail here because where multiple OS and Arch combinations target a single URI it is challenging to find the failing test case.